main.cc

```diff
@@ -560,6 +560,10 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             configurable::init_all(opts, *cfg, *ext).get();
             cfg->setup_directories();
 
+            smp::invoke_on_all([cache_index_pages = cfg->cache_index_pages()] {
+                sstables::global_cache_index_pages = cache_index_pages;
+            }).get();
```

Comment:
This won't work with LiveUpdate, unless we add a listener and re-broadcast it, or have the config accessed each time.

(or fix #7316, really overdue)